environment/src/test/java/com/sequenceiq/environment/environment/service/EnvironmentUpgradeCcmServiceTest.java

```diff
@@ -0,0 +1,85 @@
+package com.sequenceiq.environment.environment.service;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.UUID;
+
+import org.jetbrains.annotations.NotNull;
```

Comment:
Perhaps to be removed; typically not used in other classes.

Reply:
removed